debugging/coreclr/outputManager.ts

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+type outputCallback<T> = (result: T) => string;
+
+export interface OutputManager {
+    append(content: string): void;
+    appendLine(content: string): void;
+    performOperation<T>(startContent: string, operation: (outputManager: OutputManager) => Promise<T>, endContent?: string | outputCallback<T>, errorContent?: string | outputCallback<Error>): Promise<T>;
+}
+
+export class DefaultOutputManager implements OutputManager {
+    private skipFirstLine: boolean = false;
+    private isShown: boolean = false;
+
+    constructor(private readonly outputChannel: vscode.OutputChannel, private readonly level: number = 0) {
+    }
+
+    public append(content: string): void {
+        if (this.level) {
+            const split = content.split(/[\n\r]/g);
```

Comment:
On Windows this will probably result in empty entries in the split array.

Reply:
Yes, it was a naive implementation.  I've replaced it with something more robust and has unit tests.